azure-pipelines-samples.yml

```diff
@@ -256,4 +560,4 @@ steps:
     inputs:
       pathtoPublish: '$(Build.ArtifactStagingDirectory)'
       artifactName: WinMLPublicSamples-RS5
```

Comment:
Let's just change the artifactName to WinMLPublicSamples, since we aren't building against different SDKs anymore.
Since we are moving the tests into this build pipeline, there's no need to worry about changing the artifact name in the test pipeline to match.